From e97aa322238e41713c99a6ebf80392a9159f4199 Mon Sep 17 00:00:00 2001
From: Jasmin Jessich <jasmin@anw.at>
Date: Sat, 28 Jul 2018 15:02:20 +0200
Subject: [PATCH] media: i2c: fix warning in Aptina MT9V111

This fixes the "'idx' may be used uninitialized in this function"
warning.

Cc:  Jacopo Mondi <jacopo+renesas@jmondi.org>
Signed-off-by: Jasmin Jessich <jasmin@anw.at>
---
 drivers/media/i2c/mt9v111.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/mt9v111.c b/drivers/media/i2c/mt9v111.c
index da8f6ab..58d5f22 100644
--- a/drivers/media/i2c/mt9v111.c
+++ b/drivers/media/i2c/mt9v111.c
@@ -884,7 +884,7 @@ static int mt9v111_set_format(struct v4l2_subdev *subdev,
 	struct v4l2_mbus_framefmt new_fmt;
 	struct v4l2_mbus_framefmt *__fmt;
 	unsigned int best_fit = ~0L;
-	unsigned int idx;
+	unsigned int idx = 0;
 	unsigned int i;
 
 	mutex_lock(&mt9v111->stream_mutex);
-- 
2.7.4

